Engineering-Management.md

```diff
@@ -0,0 +1,63 @@
+- I can effectively resolve issues with underperformers with some guidance from my manager
+- I can confidently and proactively coach engineers up to senior (L3) level, accelerating them along their personal growth trajectory. I can coach engineers beyond senior (L3) level with support from my manager
+- I give direct feedback which is timely, actionable, and helpful. The feedback I give in performance reviews is rarely a surprise because it summarises the timely, actionable, and helpful feedback my teammate has been receiving.
+- I am learning how to think about the talent on my team strategically: how to create growth opportunities for team members while addressing gaps and risks through recruiting, training, and performance management
+- I structure my team so the right people are in the right roles with clearly defined responsibilities. In consultation with my manager, I ensure projects are staffed appropriately to set them up for success, placing my strongest technical leaders where they are the most needed and impactful
+
+### :octopus: Culture
+- I take accountability for the culture of my team
+- I keep my team focused on quality and regular releases
+- I keep my team focused on customer success
+- I help my team navigate organisational change
+- I am sensitive to the needs of my team, and I proactively help my team through difficult situations
+- I recognise dysfunction in my team and work effectively to resolve it in a timely manner.
+- I listen to different perspectives and I remove biases from my words and actions 
+- I create an inclusive environment for my team where it’s safe to speak up
+
+## L5 Senior Engineering Manager
+
+> I employ a robust managerial toolbox to independently lead a team, or teams of engineers delivering direct business impact
+
+### :triangular_flag_on_post: Leadership
+- I am able to independently identify and tackle projects that solve open-ended, ambiguous problems that are often beyond the scope of my immediate team.
+- I take accountability for the people in my team or teams, the way they work, and the results they deliver.
+- I am effective at influencing outcomes across multiple teams and with cross-functional partners, often requiring conflict resolution and managing ambiguity.
+- I identify ways that my team can work more effectively with other teams in the company and create lasting change
+- I drive impactful outcomes in the face of significant organizational and technical ambiguity and change.
+- I write and speak with clarity, tailor my message to my audience, and listen to understand others
+- I proactively share information so the right people are informed and aligned
+
+### :deciduous_tree: Talent
+- I autonomously guide my team to the sustainable delivery of business impact through my mastery of execution and people management.
+- I am sensitive to the needs of my team or teams, and I proactively help them through difficult situations
```

Comment:
This is listed under Culture for an L4, but it's made its way into the Talent section for L5. That's a bit confusing. 
I think either Culture or Talent is fine but we should be consistent between the two levels.

Reply:
Fixed